tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,156 @@
+                             
+    def test_national_heritage_day(self):
+        holidays_2022 = TurksAndCaicosIslands(years=2022)
+        self.assertIn(date(2022, 10, 10), holidays_2022)
+        self.assertEqual(holidays_2022[date(2022, 10, 10)], "National Heritage Day")
+        
+        holidays_2025 = TurksAndCaicosIslands(years=2025)
+        self.assertIn(date(2025, 10, 13), holidays_2025)
+        self.assertEqual(holidays_2025[date(2025, 10, 13)], "National Heritage Day")
+                
+    def test_national_day_of_thanksgiving(self):
+        self.assertHolidayName("National Day of Thanksgiving", (f"{year}-11-{day}" for year, day in 
```

Ruff (0.8.2)

Line too long (100 > 99)

(E501)